src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,12 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+This is a bugfix release:
+
+* Stop using cluster-owned tag for ASW Security Group lookup.
```

Comment:
This release also adds support for OpenShift 4.12, fixes https://github.com/submariner-io/lighthouse/pull/860, https://github.com/submariner-io/submariner/issues/2115, and includes some MTU changes (@yboaron what’s the exact status of 0.13.2? there were some backports and reverts).

Reply:
MTU bug fix is listed for 0.14.0 release.
I'll add a note about OpenShift 4.12 support

Reply:
The 0.13.2 release notes need to list _all_ the changes between 0.13.1 and 0.13.2. Readers can’t assume that changes made in 0.14.0 are also in 0.13.2.

Reply:
2 PRs related to MTU : 

a.  It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments, covered in this PR.
b. [ fix MTU bug for OVNK](https://github.com/submariner-io/submariner/commit/9067306e47d64775ce0838fb3985a0aadbe4cd1c) , already included in 0.13.1

So, I think that the current proposal covered all MTU issues for 0.13.2